src/software/pipeline/main_LdrToHdrMerge.cpp

```diff
@@ -175,7 +175,10 @@ int aliceVision_main(int argc, char** argv)
         }
     }
     std::vector<std::shared_ptr<sfmData::View>> targetViews;
-    hdr::selectTargetViews(targetViews, groupedViews, offsetRefBracketIndex);
+
+    const fs::path targetIndexFilepath(fs::path(inputResponsePath).parent_path() / (std::string("exposureRefIndexes.txt")));
+
+    hdr::selectTargetViews(targetViews, groupedViews, offsetRefBracketIndex, targetIndexFilepath.string());
```

Comment:
offsetRefBracketIndex should be initialized to "automatic"
`int offsetRefBracketIndex = 1000; // By default, use the automatic selection`